### ISSUE TITLE
Can't Block All Device Recognition Attempts - Settings Changes Back Automatically to Only Block Cross-Site Device Recognition Attempts

### ISSUE THREAD

**Issue by desmonk** [brave/brave-browser#8665]
## Description 

Device recognition blocking goes back to blocking only cross-site ones even when user chooses to block all attempts.

## Steps to Reproduce

   1.  chrome://settings/privacy
   2.  Device recognition
   3.  select "Block all..."
   4. Exit browser
   5. Check above option from 1 to 2
   6. Goes back to "only block cross-site..."

## Actual result:

User selected settings get changed

## Expected result:
User selected option should not get changed

## Reproduces how often: 

Every time you start Brave.

## Brave version (brave://version info)

1.3.118

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Not checked
- Can you reproduce this issue with the dev channel? Not checked
- Can you reproduce this issue with the nightly channel? Not checked

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

## Miscellaneous Information:
![IMG_067](https://user-images.githubusercontent.com/55331939/76567397-8cee5600-64a6-11ea-9a26-aa49cffe641d.gif)

**Comment by rebron** [comment]
@desmonk Do you have settings to be cleared on exit? 
Also, this is expected behavior as the global shields settings are set for shields settings for sites going forward and individual on/off settings for Shields previously set are done at the shield level.
cc: @tomlowenthal

**Comment by desmonk** [comment]
Yes, that is true. All data clear on exit. Should that create this issue though? I mean settings should remain constant irrespective of that. Sorry for late reply. Glad to see your input!

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
